### DISCUSSION TITLE
30s Segmentation

### DISCUSSION THREAD

**Discussion by catalwaysright** [openai/whisper#490] [General]
The whisper model will segment the audio into 30s chunks and then do the transcription. I am curious that how the model do this segmentation without cutting a word in middle.

**Comment by JunZhan2000** [comment]
You can read the transcribe function in transcribe.py. because whisper will output the timestamps, model will output the last full sentence at end timestamps, and then cut the audio from here to do next decoding, such as 25s-5s, not 30s -60s
